numpy/core/fromnumeric.py

```diff
@@ -2059,15 +2059,14 @@ def clip(a, a_min, a_max, out=None, **kwargs):
     ----------
     a : array_like
         Array containing elements to clip.
-    a_min : scalar or array_like or None
+    a_min : array_like or None
         Minimum value. If None, clipping is not performed on lower
         interval edge. Not more than one of `a_min` and `a_max` may be
         None.
-    a_max : scalar or array_like or None
+    a_max : array_like or None
         Maximum value. If None, clipping is not performed on upper
         interval edge. Not more than one of `a_min` and `a_max` may be
-        None. If `a_min` or `a_max` are array_like, then the three
-        arrays will be broadcasted to match their shapes.
+        None. `a_min` and  `a_max` are broadcast against `a`.
```

Comment:
I wonder if it makes sense to combine these into a single docstring,
```
a_min, a_max : array_like or None
    Minimum and maximum value. If None, clipping is not performed on the corresponding edge. Not more than one of `a_min` and `a_max` may be None. These are broadcast against `a`.

Reply:
That definitely looks better (as in easier to read). I also thought about that but was unsure whether that is OK formatting wise. I'll change that.